### ISSUE TITLE
Please add support for UI scaling

### ISSUE THREAD

**Issue by prokoudine** [sadko4u/lsp-plugins#68]
On a HiDPI display (2560x1440), the controls in the EQ interface are tiny and barely usable:

![Снимок экрана от 2020-01-08 13-02-24](https://user-images.githubusercontent.com/57467/71969351-8aa61c00-3217-11ea-9de5-c0fa223b51d0.png)

In comparison, this is what I get with x42-eq and 150% scaling (it has multiple options between 100% and 200%) :

![Снимок экрана от 2020-01-08 13-06-30](https://user-images.githubusercontent.com/57467/71969467-c345f580-3217-11ea-9a79-8028370d331c.png)

Please allow configurable scaling for UI controls.

AFAIK, there is now an LV2 extension that communicates UI scale from host to plug-in, but I could be wrong. Better ask @x42 :)

**Comment by sadko4u** [comment]
This feature will be available as a part of toolkit update in the 1.2.0 release.

**Comment by sadko4u** [comment]
Published version 0.5.0 of the new toolkit which supports widget scaling:
https://github.com/sadko4u/lsp-tk-lib

**Comment by mikefaille** [comment]
Hi, thank your for the enormous contribution on for those quality opensource plugins on Linux.

Using Catia, the Windows scaling is not the correct one by default.  : 

![Screenshot from 2021-06-21 12-31-40](https://user-images.githubusercontent.com/978196/122817358-aa56c600-d28c-11eb-890a-ad2b7979ca08.png)

Here it's my DPI and resolution according xrandr : 

```
DP-0 connected primary 3840x2160+0+0 (normal left inverted right x axis y axis) 382mm x 214mm
   3840x2160     60.02*+
```

**Comment by sadko4u** [comment]
The window scaling will be supported in 1.2.0 release, not earlier.
Sorry. I've already spend more than one year while moving towards 1.2.0.
You still need to wait a bit. But the preliminary results with the UI scaling are great (in the development repository).

**Comment by mikefaille** [comment]
Oh sorry @sadko4u  I thought it was the version 1.30 but it was the version 1.1.30 ! 🤦‍♂️ You do well please stick on your plan if it was the best as strategy it's fine then !

**Comment by sadko4u** [comment]
The current state of the new plugin framework is the following:
- All widgets now support UI scaling;
- Custom font scaling is supported, too;
- Introduced new UI styling system;
- The engine allows to dynamically change visual schemas;
- Added support of custom fonts.

Currently the work is going on:
- splitting plugins into separate repositories;
- refactoring and re-introducing the UI for plugins;
- tuning widgets and widget controllers to match the new UI design;
- adopting the build process.

**Comment by sadko4u** [comment]
Available in 1.2.0.

**Comment by mikefaille** [comment]
Thank you !
